portfolio.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/grid.css">
+    <title>Document</title>
+</head>
+
+<div class="wrapper">
+    <div class="grid-container">
+        <header class="grid-header">
+        </header>
+
+        <nav class="grid-top-nav">
+            <ul id="grid-nav-ul">
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="about"><a href="about.html">About</a></li>
+                <li id="portfolio"><a href="portfolio.html">Portfolio</a></li>
+                <li id="blog"><a href="code-journal.html">Blog</a></li>
+                <li id="contact"><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+
+        <div class="grid-main" id="hero">
+            <section class="grid-section">
+                <h1 id="portfolio-h1">Recent Projects:</h1>
+                <div class="portfolio-project-wrapper">
+                    <ol id="p-github-ul">
```

Comment:
You seem to have a lot of elements with this `id`.  Ids are supposed to be unique.  Make this a class instead maybe.